TypeCobol.Test/Utils/Comparator.cs

```diff
@@ -511,6 +511,13 @@ public override bool Visit(CommitStatement commitStatement)
                 //No SqlObject in CommitStatement
                 return true;
             }
+
+            public override bool Visit(TruncateStatement truncateStatement)
+            {
+                _writer.WriteLine($"line {truncateStatement.Line}: {nameof(truncateStatement)}");
```

Comment:
Beware you confused the name of the parameter with the name of the type, this is `nameof(TruncateStatement)`